index.html

```diff
@@ -609,22 +609,22 @@ <h2>DNS-Based Service Discovery</h2>
                         <tr class="rfc2119-table-assertion" id="introduction-dns-sd-service-name">
                             <td><code>_wot._tcp</code></td>
                             <td>Thing</td>
-                            <td>HTTP</td>
+                            <td>HTTP, HTTPS, CoAP over TCP or CoAP over TLS</td>
```

Comment:
Better to use "over TLS" consistently
```suggestion
                            <td>HTTP over TCP, HTTP over TLS/TCP, CoAP over TCP, or CoAP over TLS/TCP</td>
```